### ISSUE TITLE
When __debug flag is enabled, reading task report fails.

### ISSUE THREAD

**Issue by xsorifc28** [cathive/concourse-sonarqube-resource#61]
I am trying to perform a PR analysis. Code coverage information is missing in PR analysis when run through concourse. If I run locally, the code coverage information is there.

 I went to debug, but when I enable __debug flag I get:
```
[INFO] ------------------------------------------------------------------------/sonar/report-task.txt ]]
+ echo 'error: SonarQube report could not be located.'
error: SonarQube report could not be located.
+ exit 1
```

If I turn off the debug flag, the `put` step passes.
```Reading SonarQube scanner report (/tmp/build/put/build/target/sonar/report-task.txt)...```

**Comment by headcr4sh** [comment]
Thanks for the bug report.
Probably, the bash `+x` flag causes this issue. I assume it might lead to unexpected output on stdout when calling bash functions, but I am not sure....

**Comment by xsorifc28** [comment]
Any insight into why the coverage would be not working for PR analysis or branch analysis?

**Comment by headcr4sh** [comment]
Not (yet). I haven't had the time to look into this issue and probably won't find the time during the next few days, either. :-(

(This is an unpaid side-project, so my resources are a bit limited,... :-()

I can only *guess*, that `+x` messes up stuff that goes to stdout when calling bash functions. But that's just a theory for now....

**Comment by Etone** [comment]
As I had the same Issue just now, i might help. The Problem seems to be, that maven prints out the BUILD SUCCESS Info even with the -q flag in debug mode. This leads to a corrupted path that is not present and therefore reading the report fails. 

Path with debug enabled
```
/tmp/build/put/build-artifacts/target
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  12.703 s
[INFO] Finished at: 2020-06-25T07:13:45Z
[INFO] ------------------------------------------------------------------------/sonar/report-task.txt
```
As it looks, this looks to be from the ```+x``` flag

**Comment by xsorifc28** [comment]
Ah..so it thinks the path is `[INFO] ------------------------------------------------------------------------/sonar/report-task.txt`?

**Comment by headcr4sh** [comment]
Hopefully this issue has been fixed by #62
